docs/articles/documentation/using-testcafe/programming-interface/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: TestCafe Programming Interface
+permalink: /documentation/using-testcafe/programming-interface/
+---
+# TestCafe Programming Interface
+
+This section provides reference for the API that allows you use TestCafe from your NodeJS applications.
+
+## Summary
+
+This part outlines the workflow of using TestCafe programmatically.
+
+The first thing you need to create is the [TestCafe server](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/) instance.
+Use the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/createTestCafe/) factory function for this.
+
+```js
+const createTestCafe = require('testcafe');
+const testCafe       = await createTestCafe('localhost', 1337, 1338);
+```
+
+This is the main point of access to TestCafe. You can now use it to create other entities required to execute tests:
+the [remote browser connections](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/) (used for connecting remote devices)
+and the [test runner](/testcafe/documentation/using-testcafe/programming-interface/Runner/) (an object that controls test runs).
```

Comment:
Just style nit IMHO: put entities in the order of significance - test runner first